dpub-aam/manual/META.yml

```diff
@@ -0,0 +1,3 @@
+spec: https://w3c.github.io/dpub-aam/
+suggested_reviewers:
+  - spectranaut
```

Comment:
Reviewers, the file renaming below is just moving the manual files into a new "manual" directory. Scroll past those or collapse the dir to see the new file diff in `./role/roles.html`